packages/utils/telemetry-utils/api-report/telemetry-utils.alpha.api.md

```diff
@@ -75,7 +75,7 @@ export interface ITelemetryPerformanceEventExt extends ITelemetryGenericEventExt
 // @alpha
 export type ITelemetryPropertiesExt = Record<string, TelemetryEventPropertyTypeExt | Tagged<TelemetryEventPropertyTypeExt>>;
 
-// @alpha
+// @alpha @deprecated
```

Comment:
That's nice that test files don't seem to enforce no import deprecated 😅

Reply:
Yeah, we made an explicit exception for tests.